benchmarks/benchmark/tools/locust-load-inference/manifest-templates/locust-worker-controller.yaml.tpl

```diff
@@ -45,10 +47,8 @@ spec:
               value: ${tokenizer}
             - name: USE_BEAM_SEARCH
               value: ${use_beam_search}
-%{ for hugging_face_token_secret in hugging_face_token_secret_list ~}
             - name: HUGGINGFACE_TOKEN
               valueFrom:
                 secretKeyRef:
-                  name: hf-token
-                  key: HF_TOKEN
-%{ endfor ~}
+                  name: huggingface-secret
```

Comment:
Does this change break any of the other cases?

Reply:
oop. Reverting. This was just for my cluster setup. A

lthough, this was changed recently and now seems to be opinionated on having the secret stored in GCP as opposed to using K8s secrets. Was this intentional?

Reply:
SGTM. I'll make a note.